plugins/provisioners/salt/bootstrap-salt.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/sh -
 
+cd `mktemp --directory`
```

Comment:
Perfect! The only change here is we should use `mktemp -d` instead of the full flag, because some OS's only support the short version for some reason.